lib/users.js

```diff
@@ -19,11 +19,21 @@ function makeAdmin(user, done) {
 
     console.log('Admin status granted to: ' + user);
 
-    // if in production, notify core about new admin
+    // if in production, notify all other admins about new admin
     if (env === 'production') {
-      email.notify_new_admin(user);
+      getAdmins(function (err, admins) {
+        admins.forEach(function notifyAdmin(admin) {
+          email.notify_new_admin(user, admin.email);
+        });
+      });
     }
 
     done(null, num);
   });
 }
+
+function getAdmins(done) {
+  User.find({ account_level: 1 }, function (err, admins) {
```

Comment:
Would this also send an email to the newly created user (if it's an admin)? Seems like that user should be ignored.